### PR TITLE
CI: Sign the binaries for Endless Key USB image

### PR DESCRIPTION
**Pull request by starnight**
To sign the binaries for Endless Key USB image, split package and push to Microsoft store as anther two jobs. Also, add a new job to sign binaries with SignPath service.

https://phabricator.endlessm.com/T33599

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -196,23 +212,83 @@ jobs:
...
+
+    steps:
+      - name: Download the package
+        uses: actions/download-artifact@v3
+        with:
+          name: endlesskey.appx
+
+      - name: Release
+        uses: softprops/action-gh-release@v1
+        if: startsWith(github.ref, 'refs/tags/')
```
Comment (danigm) [inline .github/workflows/main.yml]: This can be removed

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -196,23 +212,83 @@ jobs:
...
       - name: Release
         uses: softprops/action-gh-release@v1
-        if: startsWith(github.ref, 'refs/tags/')
+        if: steps.sing_binaries.outcome == 'success'
         with:
           files: |
             kolibri-windows.zip
-            endlesskey.appx
+            kolibri-electron.signed.exe
+            Kolibri.signed.exe
```
Comment (danigm) [inline .github/workflows/main.yml]: After signing the files we should replace the original unsigned version and zip again. So the `kolibri-windows.zip` file that's uploaded to the release should contain the files signed.

The EK USB Image script just downloads the `kolibri-windows.zip` and unpacks that in the USB, so the zip should contain the signed files.

**Review by danigm — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yml (hunk 2 above)
Comment (starnight) [reply .github/workflows/main.yml]: The filenames should be `kolibri-electron.exe` and `Kolibri.exe` (signed) in the zip, or the new name `kolibri-electron.signed.exe` and `Kolibri.signed.exe`?

File: .github/workflows/main.yml (hunk 1 above)
Comment (starnight) [reply .github/workflows/main.yml]: ok!

File: .github/workflows/main.yml (hunk 2 above)
Comment (starnight) [reply .github/workflows/main.yml]: uh!  I think I understand.

**Review by danigm — APPROVED** [review]
LGTM

**Pull request merged**